src/test.rs

```diff
@@ -6,27 +6,28 @@ use codec::{Decode, Encode};
 use pact::compiler::{Contract, DataTable};
 use pact::interpreter::OpCode;
 use pact::types::{Numeric, PactType, StringLike};
-use std::string::{String, ToString};
+use std::string::String;
 use std::vec::Vec;
 
-#[test]
-fn it_works_encode() {
-    let method = Method {
-        name: "method_test".to_string(),
-        block_cooldown: None,
-        constraints: None,
-    };
+fn config_methods(method: &Method) -> Vec<(String, Method)> {
```

Comment:
nitpick: can these be called `make_methods` or `build_methods` config doesn't seem right

Reply:
No problem